src/commands/utilities.rs

```diff
@@ -0,0 +1,91 @@
+use aws_sdk_cloudformation::{
```

Comment:
since this file is only dealing w/ cloud_formation helpers, and not implementing a nitrogen command, I would suggest moving this file to `src/cf_utilities.rs`. eventually we will likely want to refactor most of the code in commands to be a rust lib instead of just CLI, and the `commands/` folder will just implement CLI stubs wrapping that lib

Reply:
That's a good idea, I moved the file!